Topics/User_Experience.md

```diff
@@ -66,7 +224,6 @@ function App() {
 
 export default App;
```

Comment:
I think the page renders the Click me as a button so users are unable to see the components, maybe wrap with <pre> tag.